pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -221,7 +234,7 @@ private boolean endCriteriaReached() {
 
   protected void consumeLoop() {
     final long _endOffset = Long.MAX_VALUE; // No upper limit on Kafka offset
-    segmentLogger.info("Starting consumption loop start offset {}, end offset {}", _currentOffset, _endOffset);
+    segmentLogger.info("Starting consumption loop start offset {}, finalOffset {}", _currentOffset, _finalOffset);
```

Comment:
The current format is easier if we are using grep/sed/awk etc., so I will prefer to keep this as is.